oxidation/libparsec/crates/protocol/tests/test_invite.rs

```diff
@@ -563,24 +563,24 @@ fn serde_invite_1_greeter_wait_peer_rep(
 }
 
 #[rstest]
-fn serde_invite_2a_claimer_send_hashed_nonce_hash_nonce_req() {
-    // Generated from Python implementation (Parsec v2.6.0+dev)
+fn serde_invite_2a_claimer_send_hashed_nonce_req() {
+    // Generated from Python implementation (Parsec v2.10.0+dev)
```

Comment:
should have used Parsec 2.6 to re-generate the example in order to be extra-careful (to avoid generating example with the rust code given we should only trust the python one)

Reply:
2.10 is still python based

Reply:
👍